tasks-by-users/mmusk/06-css-loadinganimation-task/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Loading Animation</title>
+  </head>
+  <body>
+    <div class="ring">
+      LOADING
+      <span></span>
```

Comment:
why there is not the little circle on the ring? `span`

Reply:
added @shaikh-7abish